src/pyasm/search/sql_dumper.py

```diff
@@ -349,6 +350,23 @@ def dump_tactic_inserts(my, path, mode='sql'):
         column_info = SearchType.get_column_info(my.search_type)
 
         for sobject in my.sobjects:
+
+            if relative_dir_column:
```

Comment:
if relative_dir_column: should be outside the for loop.. no need to check it for each item